src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -241,7 +246,7 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
     @PreAuthorize("hasRole('ADMIN')")
     public ResponseEntity<Void> printCompassStatisticForExercise(@PathVariable Long exerciseId) {
         ModelingExercise modelingExercise = modelingExerciseRepository.findByIdElseThrow(exerciseId);
-        compassService.printStatistic(modelingExercise.getId());
+        // compassService.printStatistic(modelingExercise.getId());
```

Comment:
So this method does nothing at the moment, is that still intended?

Reply:
Yes, it will be done in a follow up PR

Reply:
ah ok nice :+1: